### PR TITLE
aria-hidden for icon, dynamic aria-label for links

### PR DESCRIPTION
**Pull request by patrickhlauke**
currently, both links lack context (particularly the count one, being
announced simply as "65,350 link" or similar). with aria-label, these
links now make more sense to screenreader/AT users
additionally, aria-hidden on the icon ensures no AT will attempt to read
the generated content icon font character

### CODE REVIEW

**Comment by mdo** [comment]
Mind rebasing with the latest?

File: github-btn.source.html [hunk 1]
```diff
@@ -168,18 +169,23 @@
       case 'watch':
         if (v == '2') {
           counter.innerHTML = addCommas(obj.data.subscribers_count);
+          counter.setAttribute('aria-label',counter.innerHTML + ' watchers' + labelSuffix);
```
Comment (XhmikosR) [inline github-btn.source.html]: Maybe add a space after comma in your newly added lines.

**Comment by patrickhlauke** [comment]
Done, I think...

**Comment by mdo** [comment]
Thanks dude!

**Comment by patrickhlauke** [comment]
my pleasure :+1:

**Comment by patrickhlauke** [comment]
https://ghbtns.com/ is still showing the old code. am i right in thinking that the live site version isn't automagically updated with the latest repo changes?

**Comment by XhmikosR** [comment]
Yeah, that is the gh-pages branch.

**Comment by mdo** [comment]
I'll bump everything once the rest of the v3 shiz ships :).

**Pull request merged**